src/styles/card.scss

```diff
@@ -0,0 +1,153 @@
+@import './variables';
+
+.card {
+  box-sizing: border-box;
+  align-items: center;
+  width: 200px;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
```

Comment:
The font family is already defined in the `body` in the `index.scss` file. So, there is no need to define it again here. Please remove this line.